  FileName.c
D:\laba1\lab1\popitka2\FileName.c(18): warning C4715: check_sum: значение возвращается не при всех путях выполнения
  Test.cpp
D:\laba1\lab1\popitka2\FileName.c(18): warning C4715: check_sum: значение возвращается не при всех путях выполнения
Test.obj : error LNK2019: ссылка на неразрешенный внешний символ "bool __cdecl testing::internal::IsTrue(bool)" (?IsTrue@internal@testing@@YA_N_N@Z) в функции "public: static void (__cdecl*__cdecl testing::internal::SuiteApiResolver<class testing::Test>::GetSetUpCaseOrSuite(char const *,int))(void)" (?GetSetUpCaseOrSuite@?$SuiteApiResolver@VTest@testing@@@internal@testing@@SAP6AXXZPBDH@Z).
Test.obj : error LNK2019: ссылка на неразрешенный внешний символ "public: __thiscall testing::internal::GTestLog::GTestLog(enum testing::internal::GTestLogSeverity,char const *,int)" (??0GTestLog@internal@testing@@QAE@W4GTestLogSeverity@12@PBDH@Z) в функции "public: static void (__cdecl*__cdecl testing::internal::SuiteApiResolver<class testing::Test>::GetSetUpCaseOrSuite(char const *,int))(void)" (?GetSetUpCaseOrSuite@?$SuiteApiResolver@VTest@testing@@@internal@testing@@SAP6AXXZPBDH@Z).
Test.obj : error LNK2019: ссылка на неразрешенный внешний символ "public: __thiscall testing::internal::GTestLog::~GTestLog(void)" (??1GTestLog@internal@testing@@QAE@XZ) в функции "public: static void (__cdecl*__cdecl testing::internal::SuiteApiResolver<class testing::Test>::GetSetUpCaseOrSuite(char const *,int))(void)" (?GetSetUpCaseOrSuite@?$SuiteApiResolver@VTest@testing@@@internal@testing@@SAP6AXXZPBDH@Z).
Test.obj : error LNK2019: ссылка на неразрешенный внешний символ "public: __thiscall testing::Message::Message(void)" (??0Message@testing@@QAE@XZ) в функции "private: virtual void __thiscall TestCaseName_TestName_Test::TestBody(void)" (?TestBody@TestCaseName_TestName_Test@@EAEXXZ).
Test.obj : error LNK2019: ссылка на неразрешенный внешний символ "class testing::AssertionResult __cdecl testing::AssertionSuccess(void)" (?AssertionSuccess@testing@@YA?AVAssertionResult@1@XZ) в функции "class testing::AssertionResult __cdecl testing::internal::CmpHelperEQ<int,int>(char const *,char const *,int const &,int const &)" (??$CmpHelperEQ@HH@internal@testing@@YA?AVAssertionResult@1@PBD0ABH1@Z).
Test.obj : error LNK2019: ссылка на неразрешенный внешний символ "class testing::AssertionResult __cdecl testing::internal::EqFailure(char const *,char const *,class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &,class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &,bool)" (?EqFailure@internal@testing@@YA?AVAssertionResult@2@PBD0ABV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@1_N@Z) в функции "class testing::AssertionResult __cdecl testing::internal::CmpHelperEQFailure<int,int>(char const *,char const *,int const &,int const &)" (??$CmpHelperEQFailure@HH@internal@testing@@YA?AVAssertionResult@1@PBD0ABH1@Z).
Test.obj : error LNK2019: ссылка на неразрешенный внешний символ "class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > __cdecl testing::internal::GetBoolAssertionFailureMessage(class testing::AssertionResult const &,char const *,char const *,char const *)" (?GetBoolAssertionFailureMessage@internal@testing@@YA?AV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@ABVAssertionResult@2@PBD11@Z) в функции "private: virtual void __thiscall TestCaseName_TestName_Test::TestBody(void)" (?TestBody@TestCaseName_TestName_Test@@EAEXXZ).
Test.obj : error LNK2019: ссылка на неразрешенный внешний символ "void const * __cdecl testing::internal::GetTestTypeId(void)" (?GetTestTypeId@internal@testing@@YAPBXXZ) в функции "void __cdecl `dynamic initializer for 'private: static class testing::TestInfo * TestCaseName_TestName_Test::test_info_''(void)" (??__E?test_info_@TestCaseName_TestName_Test@@0QAVTestInfo@testing@@A@@YAXXZ).
Test.obj : error LNK2019: ссылка на неразрешенный внешний символ "class testing::TestInfo * __cdecl testing::internal::MakeAndRegisterTestInfo(char const *,char const *,char const *,char const *,struct testing::internal::CodeLocation,void const *,void (__cdecl*)(void),void (__cdecl*)(void),class testing::internal::TestFactoryBase *)" (?MakeAndRegisterTestInfo@internal@testing@@YAPAVTestInfo@2@PBD000UCodeLocation@12@PBXP6AXXZ3PAVTestFactoryBase@12@@Z) в функции "void __cdecl `dynamic initializer for 'private: static class testing::TestInfo * TestCaseName_TestName_Test::test_info_''(void)" (??__E?test_info_@TestCaseName_TestName_Test@@0QAVTestInfo@testing@@A@@YAXXZ).
Test.obj : error LNK2019: ссылка на неразрешенный внешний символ "public: virtual __thiscall testing::Test::~Test(void)" (??1Test@testing@@UAE@XZ) в функции "public: virtual __thiscall TestCaseName_TestName_Test::~TestCaseName_TestName_Test(void)" (??1TestCaseName_TestName_Test@@UAE@XZ).
Test.obj : error LNK2019: ссылка на неразрешенный внешний символ "protected: __thiscall testing::Test::Test(void)" (??0Test@testing@@IAE@XZ) в функции "public: __thiscall TestCaseName_TestName_Test::TestCaseName_TestName_Test(void)" (??0TestCaseName_TestName_Test@@QAE@XZ).
Test.obj : error LNK2001: неразрешенный внешний символ "protected: virtual void __thiscall testing::Test::SetUp(void)" (?SetUp@Test@testing@@MAEXXZ).
Test.obj : error LNK2001: неразрешенный внешний символ "protected: virtual void __thiscall testing::Test::TearDown(void)" (?TearDown@Test@testing@@MAEXXZ).
Test.obj : error LNK2019: ссылка на неразрешенный внешний символ "public: static class testing::UnitTest * __cdecl testing::UnitTest::GetInstance(void)" (?GetInstance@UnitTest@testing@@SAPAV12@XZ) в функции "int __cdecl RUN_ALL_TESTS(void)" (?RUN_ALL_TESTS@@YAHXZ).
Test.obj : error LNK2019: ссылка на неразрешенный внешний символ "public: int __thiscall testing::UnitTest::Run(void)" (?Run@UnitTest@testing@@QAEHXZ) в функции "int __cdecl RUN_ALL_TESTS(void)" (?RUN_ALL_TESTS@@YAHXZ).
Test.obj : error LNK2019: ссылка на неразрешенный внешний символ "void __cdecl testing::InitGoogleTest(int *,char * *)" (?InitGoogleTest@testing@@YAXPAHPAPAD@Z) в функции _main.
Test.obj : error LNK2019: ссылка на неразрешенный внешний символ "public: __thiscall testing::internal::AssertHelper::AssertHelper(enum testing::TestPartResult::Type,char const *,int,char const *)" (??0AssertHelper@internal@testing@@QAE@W4Type@TestPartResult@2@PBDH1@Z) в функции "private: virtual void __thiscall TestCaseName_TestName_Test::TestBody(void)" (?TestBody@TestCaseName_TestName_Test@@EAEXXZ).
Test.obj : error LNK2019: ссылка на неразрешенный внешний символ "public: __thiscall testing::internal::AssertHelper::~AssertHelper(void)" (??1AssertHelper@internal@testing@@QAE@XZ) в функции "private: virtual void __thiscall TestCaseName_TestName_Test::TestBody(void)" (?TestBody@TestCaseName_TestName_Test@@EAEXXZ).
Test.obj : error LNK2019: ссылка на неразрешенный внешний символ "public: void __thiscall testing::internal::AssertHelper::operator=(class testing::Message const &)const " (??4AssertHelper@internal@testing@@QBEXABVMessage@2@@Z) в функции "private: virtual void __thiscall TestCaseName_TestName_Test::TestBody(void)" (?TestBody@TestCaseName_TestName_Test@@EAEXXZ).
C:\Users\andre\OneDrive\Рабочий стол\popitka2\gtest_x64-windows\debug\lib\gtest.lib : warning LNK4272: тип компьютера библиотеки "x64" конфликтует с типом целевого компьютера "x86"
D:\laba1\lab1\popitka2\Debug\popitka2.exe : fatal error LNK1120: неразрешенных внешних элементов: 19
